docs/reference/argocd.md

```diff
@@ -73,6 +73,7 @@ Name | Default | Description
 --- | --- | ---
 Image | `quay.io/argocdapplicationset/argocd-applicationset` | The container image for the ApplicationSet controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
 Version | *(recent ApplicationSet version)* | The tag to use with the ApplicationSet container image.
+LogLevel | info | The log level to be used by the ArgoCD Application Controller component. Valid options are debug, info, error, and warn.
```

Comment:
Resources field also seem to be missing. Can you please create an entry for that as well ?

https://github.com/argoproj-labs/argocd-operator/blob/1184f4b65f628387b02b5755374e07b328a5737d/deploy/crds/argoproj.io_argocds_crd.yaml#L50

Reply:
This should be resolved now.